### PR TITLE
chore!: remove unused oauth setting

### PR DESCRIPTION
**Pull request by hugocostadev**
## Proposed changes (including videos or screenshots)

The settings  `Accounts_OAuth_Custom_Button_Label_Color` , `Accounts_OAuth_Custom_Button_Color` , `*_button_label_color` and `*_button_color`  was not being used anymore for oauth buttons

## Further comments

https://rocketchat.atlassian.net/browse/SUP-395

### CODE REVIEW

**Review by elfajriharsuda — APPROVED** [review]

**Review by ggazzo — CHANGES_REQUESTED** [review]
how about a migration to delete? otherwise old instances will show the setting with no translations

**Comment by hugocostadev** [comment]
> how about a migration to delete? otherwise old instances will show the setting with no translations

Good idea! I'll work on that! Thanks 🚀

File: apps/meteor/server/startup/migrations/v305.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	version: 305,
+	async up() {
+		const customOauthServicesButtonColors = await Settings.find(
+			{ _id: /Accounts_OAuth_.+button.+color$/ },
+			{ projection: { _id: 1 } },
+		).toArray();
+
+		for await (const setting of customOauthServicesButtonColors) {
+			await Settings.removeById(setting._id);
+		}
```
Comment (ggazzo) [inline apps/meteor/server/startup/migrations/v305.ts:16]: why not just a `deleteMany`?

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/server/startup/migrations/v305.ts (hunk 1 above)
Comment (hugocostadev) [reply apps/meteor/server/startup/migrations/v305.ts]: Much better indeed! Thanks

**Review by ggazzo — DISMISSED** [review]

**Review by debdutdeb — DISMISSED** [review]

**Review by ggazzo — APPROVED** [review]

**Pull request merged**